src/common/QuirksManager.cpp

```diff
@@ -184,6 +184,17 @@ bool QuirksManager::isConversionNeeded(
         LOGI("QuirksManager::%s() forcing internal format to I16 for low latency", __func__);
     }
 
+    // Add quirk for float output on API <21
+    if (isFloat
+            && !isInput
+            && getSdkVersion() < __ANDROID_API_L__ && builder.isFormatConversionAllowed()) {
```

Comment:
I think it is more clear to have each && on its own line.

Reply:
ha, I do too, for some reason I just missed the last one - good spot, thanks.